src/titiler/application/titiler/application/main.py

```diff
@@ -40,14 +41,34 @@
     # Try backported to PY<39 `importlib_resources`.
     from importlib_resources import files as resources_files  # type: ignore
 
+LEVELS = {
+    "debug": logging.DEBUG,
+    "info": logging.INFO,
+    "warning": logging.WARNING,
+    "error": logging.ERROR,
+    "critical": logging.CRITICAL,
+}
+# Remove any AWS-injected logger handlers to fix Lambda logging to CloudWatch
+# https://stackoverflow.com/a/45624044
```

Comment:
Was this issue caused by the upgrade or latent in our deployed titiler?

Reply:
This is not a new contribution. It's actually a common contribution to Sentera's Python projects, as we've historically had trouble getting the Python Lambda logger to work with CloudWatch, probably due to the details in that StackOverflow link.

I can try getting rid of this if you'd like.

Reply:
This doesn't matter to me a lot. If you want to take a swing at it, I'm happy to review the solution.

Reply:
Ok, I'll leave this for a python project that has better CI/CD and isn't such a pain to build/deploy/test. It's worth pushing on this at some point to make sure it's still necessary.